powerauth-push-server/src/main/java/io/getlime/push/service/DeviceRegistrationService.java

```diff
@@ -69,25 +69,35 @@ public void createOrUpdateDevice(final CreateDeviceRequest requestObject, final
         final String activationId = requestObject.getActivationId();
 
         final List<PushDeviceRegistrationEntity> devices = lookupDeviceRegistrations(appId, activationId, pushToken);
-        final PushDeviceRegistrationEntity device;
+        PushDeviceRegistrationEntity device = null;
```

Comment:
🤔 IMHO, it could be better to keep the _final_ variable_, which of course requires some code changes.

Reply:
I do not know how to achieve this easily.